ee/features/storage/s3-store.ts

```
@@ -0,0 +1,196 @@
+      this.client = this.euClient;
+      this.bucket = this.euConfig.bucket;
+    }
+  }
+
+  // Override key S3Store methods to ensure correct region
+  async create(upload: Upload): Promise<Upload> {
+    try {
+      await this.ensureCorrectRegion(upload.id);
+      return await super.create(upload);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)